openmetadata-ui/src/main/resources/ui/src/components/AddGlossary/AddGlossary.component.tsx

```diff
@@ -0,0 +1,264 @@
+  const [description] = useState<string>('');
+  const [showRevieweModal, setShowRevieweModal] = useState(false);
+  const [reviewer, setReviewer] = useState<Array<FormatedUsersData>>([]);
+
+  const onReviewerModalCancel = () => {
+    setShowRevieweModal(false);
+  };
+
+  const handleReviewerSave = (reviewer: Array<FormatedUsersData>) => {
+    setReviewer(reviewer);
+    onReviewerModalCancel();
+  };
+
+  const handleValidation = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>
+  ) => {
+    if (!allowAccess) {
+      return;
+    }
+    const value = event.target.value;
+    const eleName = event.target.name;
+    let { name } = cloneDeep(showErrorMsg);
+
+    switch (eleName) {
+      case 'name': {
+        setName(value);
+        name = false;
+
+        break;
+      }
+    }
+    setShowErrorMsg((prev) => {
+      return { ...prev, name };
+    });
+  };
+
+  const handleOptionRemove = (id: string) => {
+    setReviewer((pre) => pre.filter((option) => option.id !== id));
+  };
+
+  const validateForm = () => {
+    const errMsg = {
+      name: !name.trim(),
+    };
+    setShowErrorMsg(errMsg);
+
+    return !Object.values(errMsg).includes(true);
+  };
+
+  const handleSave = () => {
+    if (validateForm()) {
+      const data: CreateGlossary = {
+        name,
+        displayName: name,
+        description: markdownRef.current?.getEditorContent() || undefined,
+        reviewers: reviewer.map((d) => ({ id: d.id, type: d.type })),
+        owner: {
+          id: getCurrentUserId(),
+          type: 'user',
+        },
+      };
+
+      onSave(data);
+    }
+  };
+
+  const getSaveButton = () => {
+    return allowAccess ? (
+      <>
+        {saveState === 'waiting' ? (
+          <Button
+            disabled
+            className="tw-w-16 tw-h-10 disabled:tw-opacity-100"
+            size="regular"
+            theme="primary"
+            variant="contained">
+            <Loader size="small" type="white" />
+          </Button>
+        ) : saveState === 'success' ? (
+          <Button
+            disabled
+            className="tw-w-16 tw-h-10 disabled:tw-opacity-100"
+            size="regular"
+            theme="primary"
+            variant="contained">
+            <i aria-hidden="true" className="fa fa-check" />
+          </Button>
+        ) : (
+          <Button
+            className={classNames('tw-w-16 tw-h-10', {
+              'tw-opacity-40': !allowAccess,
+            })}
+            data-testid="save-webhook"
+            size="regular"
+            theme="primary"
+            variant="contained"
+            onClick={handleSave}>
+            Save
+          </Button>
+        )}
+      </>
+    ) : null;
+  };
+
+  const fetchRightPanel = () => {
+    return (
+      <>
+        <h6 className="tw-heading tw-text-base">Configure Your Glossary</h6>
+        <div className="tw-mb-5">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```

Comment:
@aashitk fix this text

Reply:
Need to get proper content for this, will be fixed in next patch